app/services/puppetdb.rb

```diff
@@ -1,8 +1,8 @@
 module Puppetdb
   API_VERSIONS = {
-    '4' => 'v4: PuppetDB 2.3, 3.0, 3.1, 3.2, 4.0 (PE 3.8, 2015.2, 2015.3)',
-    '3' => 'v3: PuppetDB 1.5, 1.6 (PE 3.1, 3.2, 3.3)',
-    '2' => 'v2: PuppetDB 1.1, 1.2, 1.3, 1.4'
+    '4' => 'v4: PuppetDB 4.0, 4.1, 4.2, 4.3',
```

Comment:
also 5.x? https://docs.puppet.com/puppetdb/5.0/api/index.html#version-4

Reply:
I haven't tested 5.x so I don't want to list it officially.